docs/docs-beta/docs/guides/dagster-service.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Deploy Dagster as a service"
+description: "Learn how to deploy Dagster as a service on a single machine"
+---
+
+# Deploy Dagster as a service
+
+This guide will walk you through deploying Dagster as a service on a single machine. It includes instructions for setting up the Dagster webserver and daemon. This approach is suitable for small-scale deployments or for testing purposes. For production environments, consider using containerized deployments or cloud-based solutions
+
+## Running the Dagster Webserver
+
+The Dagster webserver is the core component of any Dagster deployment. It serves the Dagster UI and responds to GraphQL queries.
+
+### Installation
+
+First, install the Dagster webserver:
+
+```bash
+pip install dagster-webserver
+```
+
+### Starting the Dagster Webserver
+
+To run the webserver, use the following command:
+
+```bash
+DAGSTER_HOME=/opt/dagster/dagster_home dagster-webserver -h 0.0.0.0 -p 3000
+```
+
+This configuration will:
+- Set the `DAGSTER_HOME` environment variable, which tells Dagster where to store its persistent data and logs
+- Write execution logs to `$DAGSTER_HOME/logs`
+- Listen on `0.0.0.0:3000`
+
+## Running the Dagster Daemon
+
+The Dagster daemon is necessary if you're using schedules, sensors, backfills, or want to set limits on the number of runs that can be executed simultaneously.
+
+### Installation
+
+Install the Dagster daemon:
+
+```bash
+pip install dagster
+```
+
+### Starting the Daemon
+
+Run the Dagster daemon with this command:
+
+```bash
+DAGSTER_HOME=/opt/dagster/dagster_home dagster-daemon run
```

Comment:
This was happening for `dagster-webserver` too, but when _not_ in the `dagster_home` directory, I would get:
```
$ DAGSTER_HOME=/tmp/dagster/dagster_home dagster-daemon run
Usage: dagster-daemon run [OPTIONS]
Try 'dagster-daemon run --help' for help.

Error: No arguments given and no [tool.dagster] block in pyproject.toml found.
```
But after changing to that directory it worked fine.